app/controllers/api/v1/animals_controller.rb

```diff
@@ -7,6 +7,7 @@ def show
     animal = Animal.find(params[:id])
     response["animal"] = animal
     response["reviews"] = animal.reviews
+    response["current_user_id"] = session[:current_user_id]
     render json: response
```

Comment:
this is all code smell for not having a serializer! haha you'll find that using a serializer is going to earn you dividends in terms of time spent getting all of the data ready for these json responses

this isnt necessarily grounds for rejection, but since you have to make updates anyway, you plan to either add a serializer on this PR, or the next